packages/server/src/fhir/lookups/reference.ts

```diff
@@ -75,6 +224,15 @@ export class ReferenceTable extends LookupTable {
   }
 }
 
+/**
+ * Creates a hash string for a reference row for efficient comparison.
+ * @param row - The reference table row.
+ * @returns A hash string combining resourceId, targetId, and code.
+ */
+function hashRow(row: ReferenceTableRow): string {
+  return `${row.resourceId}|${row.targetId}|${row.code}`;
```

Comment:
Could this refer to `allColumnNames`, to ensure that if new columns are added they automatically become part of the hash?